evm_arithmetization/src/cpu/kernel/interpreter.rs

```diff
@@ -857,8 +857,6 @@ impl<F: Field> State<F> for Interpreter<F> {
 
         if registers.is_kernel {
             log_kernel_instruction(self, op);
-        } else {
-            log::debug!("User instruction: {:?}", op);
```

Comment:
I don't think we should remove it. This removes everything user related, which can be useful when debugging (especially on the `zero-bin` side when we're debugging blocks and just simulating the CPU with the interpreter.

Reply:
Ah yes, it should call `Self::log_debug` instead. Don't know why I removed it.